src/metabase/api/dashboard.clj

```diff
@@ -88,26 +107,36 @@
                    (not= embedding_params (:embedding_params dash))))
       (check-embedding-enabled)
       (check-superuser)))
-  (check-500 (-> (assoc dashboard :id id)
-                 (dashboard/update-dashboard! *current-user-id*))))
-
-
+  (check-500
+   (db/update! Dashboard id
+     ;; description, position are allowed to be `nil`. Everything else must be non-nil
+     (u/select-keys-when dashboard
+       :present #{:description :position}
```

Comment:
`position` is `nil` by default unless somebody sets it. So if position is `nil` you'll just have to fall back to default sorting for the Dashboards (whatever that is) until somebody starts reördering them